scheduler/cups-driverd.cxx

```diff
@@ -183,6 +191,11 @@ static int		read_tar(cups_file_t *fp, char *name, size_t namesize,
 			         struct stat *info);
 static regex_t		*regex_device_id(const char *device_id);
 static regex_t		*regex_string(const char *s);
+void              *cupsFileGetsCall(void *input);
```

Comment:
I'm not sure why these functions have a "cupsFile" prefix - they aren't part of libcups or the cups_file_t API.